.github/workflows/ci-ultra-tests.yml

```diff
@@ -1,10 +1,6 @@
 name: ULTRA CI
 
-on:
-  push:
-    branches:
-      - disabled
-
+on: [push]
 
 jobs:
   test:
```

Comment:
Include the following to avoid duplicated GitHub Actions. See #524 .
```yml
test:
  runs-on: ubuntu-18.04
  if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
  container: huaweinoah/smarts:v0.4.3-pre
```